test/wdio.sauce.js

```diff
@@ -1,4 +1,4 @@
-var config = require('./wdio.conf').config;
+var config = require('./wdio.quick').config;
```

Comment:
That was purposeful. I thought we only wanted the data tests to be run on demand, and not part of the normal test cycle. I'd be happy to switch it back, though.